cEP-0010.md

```diff
@@ -140,16 +152,24 @@ Here is the list of aspects to be implemented along with their corresponding
   in your files given `Root.Formatting.Quotation.preferred_quotation`.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
   user's preferred quotation.
+- `Root.Formatting.Quotation.force_preferred_quotation`: This taste represents
```

Comment:
how is this different from `preferred_quotation`?

This needs a _lot_ of documentation to explain why it is different, and when it should be used, as compared to just `preferred_quotation`.

A real example would be very helpful.

Perhaps part of the problem is ... what is preferred_quotation? and
why is it not mandatory to follow the preferred quotation?

perhaps "preferred_quotation" needs to be renamed to "quotation_marker"